controller-server/src/main/java/com/yahoo/vespa/hosted/controller/deployment/InternalStepRunner.java

```diff
@@ -455,6 +456,22 @@ private boolean containersAreUp(ApplicationId id, ZoneId zoneId, DualLogger logg
         return true;
     }
 
+    /** Returns true iff all containers in the tester deployment give 100 consecutive 200 OK responses on /status.html. */
+    private boolean testerContainersAreUp(ApplicationId id, ZoneId zoneId, DualLogger logger) {
+        if (useConfigServerForTesterAPI(zoneId)) {
+            DeploymentId deploymentId = new DeploymentId(id, zoneId);
+            if (controller.jobController().cloud().testerReady(deploymentId)) {
```

Comment:
Should this use the config server way as well?

Reply:
It does (if argument is deployment id the config server will be used)